src/app/common-ui/Popup/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import styles from './style.scss';
+import PropTypes from 'prop-types';
+
+import CloseButton from '../../../assets/images/close-button.png';
+
+class Popup extends Component {
+  render() {
+    const { closePopup, titleContent, mainContent, contentBtn, linkTo, isSuccess } = this.props;
+    return (
+      <div className={styles.overlay}>
+        <div className={styles.container}>
+          <div className={styles.titleBar}>
+            <h5
+              className={`${styles.titleContent} ${
+                isSuccess === '1' ? styles.titleColorSuccess : styles.titleColorFailed
```

Comment:
Set color based on `variant`.

Use a default color in case that `variant` is not set